adapters/rack/lib/opentelemetry/adapters/rack/adapter.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Rack
+      # The Adapter class contains logic to detect and install the Rack
+      # instrumentation adapter
+      class Adapter < OpenTelemetry::Instrumentation::Adapter
+        install do |config|
+          require_dependencies
+
+          retain_middleware_names if config[:retain_middleware_names]
+          configure_default_quantization
+
+          config[:application]&.use Middlewares::TracerMiddleware
```

Comment:
I think we should leave the `.use` call to the user. I don't see this in dd-trace and I think it could cause issues if the user calls `use` for this middleware and we also call it behind the scenes.